### ISSUE TITLE
Unhandled Exception: FormatException: <version>

### ISSUE THREAD

**Issue by Muhammed-Rahif** [MostafaSolimanMO/once#20]
`Once.runOnNewVersion` causing an error on release version of flutter apps, flutter log:
```
E/flutter ( 7257): [ERROR:flutter/runtime/dart_vm_initializer.cc(41)] Unhandled Exception: FormatException: 100-beta12
E/flutter ( 7257): #0      num.parse (dart:core/num.dart:574)
E/flutter ( 7257): #1      OnceRunner.runOnNewVersion (package:once/src/runner/runner.dart:166)
E/flutter ( 7257): <asynchronous suspension>
E/flutter ( 7257): 
```
- The `100-beta12` word in the log is my app version without `.` (`1.0.0-beta.12`).

I have using the `once: ^1.5.1` before, now when upgraded it to `once: ^1.5.3` this error happend. I found out that the cause for this error is something with `package_info_plus: ^4.1.0`.

Working versions:
```
once: ^1.5.1
package_info_plus: ^3.1.2
```

Not working:
```
once: ^1.5.3
package_info_plus: ^4.1.0
```

_Some breaking changes happend to `package_info_plus`._

**Comment by MostafaSolimanMO** [comment]
Thanks for reporting it, Muhammed. Really appreciated.
I will look into it and fix it.

**Comment by MostafaSolimanMO** [comment]
Fixed on [1.5.4](https://github.com/MostafaSolimanMO/once/releases/tag/1.5.4).
Thanks @Muhammed-Rahif
